JWBestPracticeApps/FeedTableViewController/FeedTableViewController/Feed/FeedViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  FeedViewModel.swift
+//  FeedTableViewControllerV4
+//
+//  Created by Amitai Blickstein on 5/2/22.
+//
+
+import Foundation
+import JWPlayerKit
+
+/// All hard-coded and dynamic values required by the table view.
+/// Since there is only one d-source for a table view, this is a singleton.
+final class FeedViewModel {
+    // Singleton pattern
+    static let shared = FeedViewModel()
+    private init() {}
+    
+    // Dynamic values
+    private var items = [JWPlayerItem]()
+    
+    // Derived values
+    var count: Int { items.count }
```

Comment:
"Derived values" doesn't tell someone what this is. I would label this "The number of player items in the feed."

Reply:
Incorporated